### DISCUSSION TITLE
Error when trying to copy an ID in the Console

### DISCUSSION THREAD

**Discussion by seracoder** [appwrite/appwrite#4792] [Q&A] [answered]
### 👟 Reproduction steps

create a new project on a new appwrite instance and try to copy project id

### 👍 Expected behavior

copy project id

### 👎 Actual Behavior

returns error
![image](https://user-images.githubusercontent.com/41739275/204006512-b972dff2-59c9-4f29-a144-8be562954a51.png)

### 🎲 Appwrite version

Version 1.1.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment, marked as answer]
@randomloops would you please try to open the Appwrite Console using HTTPS?

**Reply by seracoder** [reply]
it worked on https.
